feature_details.md

```diff
@@ -0,0 +1,174 @@
+
+* `sharedElement` : A reference to an element in the previous Document, explained in Referencing Elements (#referencing-elements).
+
+If one of `sharedElement` or `element` is set, this is an enter or exit transition respectively. The animations for each animatable property are defined based on `transitionType`. If both `sharedElement` and `element` are set, this is a paired transition. This transition first creates a combined image using painted content from both elements based on the `content` animation specification under [Animatable Properties](#animatable-properties). Other animatable properties are then applied to this combined output.
+
+An [open request](https://github.com/WICG/shared-element-transitions/issues/28) here is the ability to add low-level customization to these animations by specifying a delay, duration, easing curve, keyframes similar to the [Web Animation API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API). A potential option to provide this capability in future iterations is to add a `keyframes` field to this dictionary which supports property types listed under [Animatable Properties](#animatable-properties). The syntax is defined under [Processing a keyframes argument](https://drafts.csswg.org/web-animations-1/#processing-a-keyframes-argument) in Web Animations API. It’s worth mentioning that the keyframes syntax already has a concept of an implicit start/end keyframe which could map to the start/end state for each property computed in the previous/new Document respectively.
+
+#### Referencing Elements
+Specifying a transition above requires the new Document to reference elements from the previous Document. We address this problem, along with flexibility to exchange contextual information between Documents, by introducing a placeholder for shared elements captured from the previous Document. The documentTransition APIs mentioned in the following explanation are clarified under [Control Flow](#control-flow).
+
+* The dictionary used to specify a shared element cached from the previous Document in documentTransition.setSameOriginTransitions has the following fields :
+    * `element` : A reference to the [Element](https://dom.spec.whatwg.org/#interface-element) in the previous Document.
+    * `propertyMap` : A `record<DOMString, any>` provided by the developer to pass any opaque contextual information to the new Document. The value for this map can be any javascript object supported by the [structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm).
+    * `metadataOnly` : A boolean indicating whether the UA should skip capturing the element snapshot, defaults to false. This is purely a performance optimization when a visually identical element is present in both Documents, which would be a common scenario for paired transitions.
```

Comment:
I'm not sure about the order of events here. I thought this API is happening on the incoming page _after_ the visual contents of the outgoing element have been captured. So, is this specified by the incoming page or the outgoing? In either case, I think this API is complicated enough that we can leave this off for now, especially if it's only an optimization. We can certainly add it later